frontend/src/metabase/core/components/MarkdownPreview/MarkdownPreview.tsx

```diff
@@ -17,24 +18,38 @@ export const MarkdownPreview = ({
   children,
   className,
   tooltipMaxWidth,
-}: Props) => (
-  <Tooltip
-    maxWidth={tooltipMaxWidth}
-    placement="bottom"
-    tooltip={
-      <Markdown disallowHeading unstyleLinks>
-        {children}
-      </Markdown>
-    }
-  >
-    <div>
-      <TruncatedMarkdown
-        allowedElements={ALLOWED_ELEMENTS}
-        className={className}
-        unwrapDisallowed
-      >
-        {children}
-      </TruncatedMarkdown>
-    </div>
-  </Tooltip>
-);
+}: Props) => {
+  const { isTruncated, ref } = useIsTruncated();
+
+  const setReactMarkdownRef: LegacyRef<HTMLDivElement> = divRef => {
+    /**
+     * react-markdown API does not allow passing ref to the container div.
+     * We can acquire the reference through its parent.
+     */
+    const reactMarkdownRoot = divRef?.querySelector("div");
```

Comment:
btw, does it always a `div` or should we get the first element?

Reply:
Always a div: https://github.com/remarkjs/react-markdown/blob/main/lib/react-markdown.js#L120

Reply:
Yeah, in our case it's always a div because we always have `className`: https://github.com/remarkjs/react-markdown/blob/main/lib/react-markdown.js#L120

Reply:
Updated to check for first element for extra safety: c9ca931eec164aae549667cc127a494226253c9a